src/Microsoft.ML.Core/Data/ISchemaBindableMapper.cs

```diff
@@ -61,15 +66,20 @@ public interface ISchemaBoundRowMapper : ISchemaBoundMapper, IRowToRowMapper
     /// return a subset of the input columns.
     /// This interface is similar to <see cref="ISchemaBoundRowMapper"/>, except it does not have any input role mappings,
     /// so to rebind, the same input column names must be used.
-    /// Implementing of this object are typically created using a definie input <see cref="ISchema"/>.
+    /// Implementation of this interface are typically created using a defined input <see cref="ISchema"/>.
```

Comment:
>Implementation of this interface are [](start = 8, length = 36)

We could have "An implementation of this interface is created using a defined input `Schema`." or "Implementations of this interface are typically created over defined input `Schema`s.`, but we cannot I think have "Implementation (singular) of this interface are (plural) typically created using a (singular) defined input `Schema`."

Reply:
Fixed. Thank you.

---
In reply to: [237587728](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237587728) [](ancestors = 237587728)